solver/src/settlement_submission.rs

```diff
@@ -0,0 +1,50 @@
+use std::time::Duration;
+
+use crate::settlement::Settlement;
+use anyhow::{anyhow, Context, Result};
+use contracts::GPv2Settlement;
+use ethcontract::GasPrice;
+use gas_estimation::GasPriceEstimating;
+use primitive_types::U256;
+
+const MAX_GAS: u32 = 8_000_000;
```

Comment:
This is a v1 thing right? I guess it applies anywhere.

Reply:
It was a constant in the gpv2 code somewhere already.